### PR TITLE
Register new package PhantomJS.jl v0.0.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [fredo-dedup/PhantomJS.jl](https://github.com/fredo-dedup/PhantomJS.jl)
Release: [v0.0.1](https://github.com/fredo-dedup/PhantomJS.jl/releases/tag/v0.0.1)
Travis: [Travis Build Status](https://travis-ci.org/fredo-dedup/PhantomJS.jl/builds/216388106)
cc: @fredo-dedup

### CODE REVIEW

File: PhantomJS/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+BinDeps
+Compat 0.8
```
Comment (tkelman) [inline PhantomJS/versions/0.0.1/requires]: looks like you might be able to get away without Compat entirely?

Comment (fredo-dedup) [inline PhantomJS/versions/0.0.1/requires]: You're right. I removed it. BinDeps requires Compat but I don't need to mention it here.

Comment (tkelman) [inline PhantomJS/versions/0.0.1/requires]: also note https://github.com/fredo-dedup/PhantomJS.jl/pull/1, packages shouldn't use `Pkg.dir` to refer to their own locations

Comment (tkelman) [inline PhantomJS/versions/0.0.1/requires]: should also remove `using Compat` from `deps/build.jl`, it's not guaranteed that BinDeps will always depend on Compat

**Pull request merged**